FEXCore/Source/Interface/Core/JIT/JIT.cpp

```diff
@@ -85,17 +86,16 @@ void Arm64JITCore::Op_Unhandled(const IR::IROp_Header* IROp, IR::NodeID Node) {
 #endif
   } else {
     auto FillF80Result = [&]() {
+      auto ResultReg = ARMEmitter::VReg::v0;
```

Comment:
You can just use VTMP1 always here, the point of TMP_ABIARGS is to explicitly indicate that such is valid